app/lib/aether/languages/language.coffee

```diff
@@ -31,7 +33,21 @@ module.exports = class Language
 
   # Return an array of UserCodeProblems detected during linting.
   lint: (rawCode, aether) ->
-    []
+    [].concat @lintUncalledMethods(rawCode, aether) # check uncalled-methods for every language
+
+  lintUncalledMethods: (rawCode, aether) ->
+    return [] unless match = rawCode.match /^ *(hero[\.|:][^\d\W]\w*)(;?)$/mi
+    problem =
+      type: 'transpile'
+      reporter: 'aether'
+      level: 'warning'
+      message: $.i18n.t('esper.do_nothing_without_parentheses', {code: match[0]})
+      hint: $.i18n.t('esper.missing_parentheses', {suggestion: "#{match[1]}()#{match[2]}"})
```

Comment:
The `aether_worker.js` worker doesn't provide the `$.i18n` global, so we either need to add it there or apply the i18n at some other point in time.

```
Uncaught TypeError: Cannot read properties of undefined (reading 't')
    at module.exports.Language.lintUncalledMethods (language.coffee?5895:44:23)
    at module.exports.Language.lint (language.coffee?5895:36:16)
    at module.exports.Aether.lint (aether.coffee?bcc5:147:5)
    at Aether.eval (aether.coffee?bcc5:113:22)
    at module.exports.Aether.hasChangedLintProblems (aether.coffee?bcc5:113:22)
    at module.exports.Aether.hasChangedSignificantly (aether.coffee?bcc5:107:39)
    at hasChangedSignificantly (aether_worker.js:45:40)
    at aether_worker.js:102:26
```